english/singapore/singapore.md

```diff
@@ -0,0 +1,6 @@
+Singapore, officially the Republic of Singapore, is a modern city-state and island country in Southeast Asia. It lies 
+off the southern tip of the Malay Peninsula and is 137 kilometres (85 mi) north of the equator; it is also separated 
+from Peninsular Malaysia by the Straits of Johor to the north, and from Indonesia's Riau Islands by the Singapore Strait 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.